### PR TITLE
Story6task1 

### PR DESCRIPTION
**Pull request by Sipho-Ndlovu**
Added the Review Component to the BookDetails component and added the CSS styling required to match the wireframe...

### CODE REVIEW

File: src/Components/BookDetail/Reviews/Reviews.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import BookReview from "./BookReview/BookReview"
+function Reviews({reviews}) {
```
Comment (Andrei-Trifu) [inline src/Components/BookDetail/Reviews/Reviews.jsx:3]: maybe add a space here as well for consistency

File: src/Components/BookDetail/Reviews/Reviews.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    return(
+        <div>
+            <h2>Reviews</h2>
+
+            {console.log(reviews)}
```
Comment (Andrei-Trifu) [inline src/Components/BookDetail/Reviews/Reviews.jsx]: remove this console.log if not needed

File: src/Components/BookDetail/Reviews/Reviews.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                {reviews.map(
+                    reviewData => 
+                        <BookReview
+                            key={reviewData.id}
+                            id={reviewData.id}
+                            name={reviewData.name}
+                            rating={reviewData.rating}
+                            review={reviewData.review}
+                        />
+
```
Comment (Andrei-Trifu) [inline src/Components/BookDetail/Reviews/Reviews.jsx]: remove this whitespace

File: src/Components/BookDetail/Reviews/Reviews.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+                )}
+            </div>
+
```
Comment (Andrei-Trifu) [inline src/Components/BookDetail/Reviews/Reviews.jsx]: remove whitespace

**Review by Andrei-Trifu — CHANGES_REQUESTED** [review]
Looks good, just a few whitespaces to remove

File: src/Components/BookDetail/Reviews/Reviews.jsx (hunk 1 above)
Comment (Sipho-Ndlovu) [reply src/Components/BookDetail/Reviews/Reviews.jsx:3]: Done, whitespace added.

File: src/Components/BookDetail/Reviews/Reviews.jsx (hunk 2 above)
Comment (Sipho-Ndlovu) [reply src/Components/BookDetail/Reviews/Reviews.jsx]: Done, console.log removed.

File: src/Components/BookDetail/Reviews/Reviews.jsx (hunk 3 above)
Comment (Sipho-Ndlovu) [reply src/Components/BookDetail/Reviews/Reviews.jsx]: Done, whitespace removed.

File: src/Components/BookDetail/Reviews/Reviews.jsx (hunk 4 above)
Comment (Sipho-Ndlovu) [reply src/Components/BookDetail/Reviews/Reviews.jsx]: Done, whitespace removed.

File: src/components/BookDetail/BookDetail.jsx [hunk 5]
```diff
@@ -63,6 +66,10 @@ function BookDetail() {
                         <p>{bookPageCount}</p>
                         <p>{bookGenre}</p>
                         <p>{bookBlurb}</p>
+
+                        <Reviews reviews={reviews} />
+
+
```
Comment (Andrei-Trifu) [inline src/components/BookDetail/BookDetail.jsx]: maybe remove some of these whitespaces as well

**Review by Andrei-Trifu — APPROVED** [review]

File: src/components/BookDetail/BookDetail.jsx (hunk 5 above)
Comment (Sipho-Ndlovu) [reply src/components/BookDetail/BookDetail.jsx]: Done, whitespaces removed.

**Pull request merged**